tests/wpt/metadata/webgl/conformance-1.0.3/conformance/attribs/gl-vertexattribpointer.html.ini

```diff
@@ -1,3 +1,3 @@
 [gl-vertexattribpointer.html]
   type: testharness
-  expected: CRASH
+  expected: OK
```

Comment:
Oh, BTW, the `expected: OK` sections can go away now. I think they're the default.